spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/TestSparkTypeConverter.java

```diff
@@ -38,6 +41,16 @@
 public class TestSparkTypeConverter {
 
   private HashMap<Type, DataType> gravitinoToSparkTypeMapper = new HashMap<>();
+  private StructType gravitinoStructType =
```

Comment:
could you make struct type test cover more cases? like nested struct type, cover different column comment, nullable

Reply:
sure

Reply:
updated.